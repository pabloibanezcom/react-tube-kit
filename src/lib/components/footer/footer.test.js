import { shallow } from 'enzyme';
import React from 'react';
import Footer from './footer';

describe('<Footer />', () => {
  it('renders three <Footer /> components', () => {
    const wrapper = shallow(<Footer text="Fake test footer" />);
    expect(wrapper.find('span').text()).toBe('Fake test footer');
    expect(wrapper).toMatchSnapshot();
  });
});
