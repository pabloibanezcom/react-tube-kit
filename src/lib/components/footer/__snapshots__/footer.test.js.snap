// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders three <Footer /> components 1`] = `ShallowWrapper {}`;
